sync/bundle/bundle_test.go

```diff
@@ -39,7 +39,8 @@ func TestMessageCompress(t *testing.T) {
 		d, _ := b.Bytes()
 		blocksData = append(blocksData, d)
 	}
-	msg := message.NewBlocksResponseMessage(message.ResponseCodeBusy, 1234, 888, blocksData, nil)
+	msg := message.NewBlocksResponseMessage(message.ResponseCodeRejected, message.ResponseCodeRejected.String(),
```

Comment:
For this particular test, `TestMessageCompress`, It's better to set the response code to OK.